tests/tests_house.py

```diff
@@ -2,10 +2,12 @@
 
 from tests.tests_tibiapy import TestTibiaPy
 from tibiapy import House
+from tibiapy.enums.tibia_enums import HouseStatus
```

Comment:
I think having an `enums.py` on the root directory of the module should be good for now, to avoid these deep imports.

-Or- add an import to the root module's `__init__.py` file, so on implementation they can be used with a shorter route.

Reply:
Done